vault/expiration.go

```diff
@@ -245,7 +245,14 @@ func expireLeaseStrategyFairsharing(ctx context.Context, m *ExpirationManager, l
 		return
 	}
 
-	m.jobManager.AddJob(job, mountAccessor)
+	queueSize := m.jobManager.AddJob(job, mountAccessor)
```

Comment:
Should we also update the gauge when an entry is removed from a queue?  Otherwise if nothing new gets enqueued we'll continue to report stale queue sizes.

Reply:
sure